### DISCUSSION TITLE
Custom Clipsnap rule in Tidclipconfig is deleting the rest of the tiddler content

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#112] [Q&A] [answered]
I use [this rule](https://scribe.tiddlyhost.com/#tweetidsnip) to emebed tweets  with custom title.
Then I click on the `dock` viewtoolbar button of that tiddler and use config in the [TidClipConfig](https://scribe.tiddlyhost.com/#TidClipConfig) to clip contents to that tiddler. Everything works correctly like snip, pin. But the [snap rule](https://scribe.tiddlyhost.com/#LocalDefaultclipSnaptitleRule) made using the clipsnap and custom image title is deleting the previously clipped contents.

**Comment by buggyj** [comment]
I have try it in you wiki and it seems to work

**Reply by arunnbabu81** [reply]
[SM34+ EvidentlyCube ExtraOperators=SM35.html.zip](https://github.com/buggyj/tiddlyclip-plugin/files/10475356/SM34%2B.EvidentlyCube.ExtraOperators.SM35.html.zip)

Can you check in this wiki. This is the local version I am using.

**Comment by buggyj** [comment, marked as answer]
you have

`$text=((*@userstring*))\n\n[img[((*@csaver(#title,@clipsnap)*))]]\n\n((*@text*))\n\n`

make it

`$text=((*$text*))((*@userstring*))\n\n[img[((*@csaver(#title,@clipsnap)*))]]\n\n((*@text*))\n\n`

**Reply by arunnbabu81** [reply]
Thank you. Its working now
